app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -192,14 +193,14 @@ private fun setupNavBarCollapsed() {
   Espresso.onView(ViewMatchers.withId(R.id.navBarCoordinator))
     .perform(ViewActions.swipeUp())
 
-  Thread.sleep(100)
+  InstrumentationRegistry.getInstrumentation().waitForIdleSync()
```

Comment:
I got annoyed by these being inconsistent screenshots & random wait times, so fixed these while I was at it